runtime/compiler/codegen/J9TreeEvaluator.cpp

```diff
@@ -734,9 +734,15 @@ uint32_t J9::TreeEvaluator::calculateInstanceOfOrCheckCastSequences(TR::Node *in
 
    TR::SymbolReference *castClassSymRef = castClassNode->getSymbolReference();
 
+   if (cg->comp()->getOption(TR_OptimizeForSpace) || cg->comp()->getOption(TR_DisableInlineCheckCast))
```

Comment:
You would need to check option `TR_DisableInlineInstanceOf` as well. 
Also there is an extra tab in the `if` block.

Reply:
Also as discussed today, Open up an issue to cleanup `TR_OptimizeForSpace`. I do not see we are setting this option any where in the code-base and also we do not provide a JIT command line option that sets this option flag.

Reply:
Already made the changes and opened the cleanup issue.